From 55ac155d42268009eaecbc183329a8744d6ae48c Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@sun.com>
Date: Tue, 7 Jul 2009 16:59:39 -0700
Subject: [PATCH] Xephyr & Xserver man page fixes

Xephyr(1): Fix quote formatting, add missing ' to contraction
Xserver(1): Add Xephyr(1) & startx(1) to SEE ALSO section

Signed-off-by: Alan Coopersmith <alan.coopersmith@sun.com>
---
 doc/Xserver.man.pre            |    4 ++--
 hw/kdrive/ephyr/Xephyr.man.pre |    6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/doc/Xserver.man.pre b/doc/Xserver.man.pre
index 12b4cd4..6154191 100644
--- a/doc/Xserver.man.pre
+++ b/doc/Xserver.man.pre
@@ -581,13 +581,13 @@ Security: \fIXsecurity\fP(__miscmansuffix__), \fIxauth\fP(1), \fIXau\fP(1),
 \fIxdm\fP(1), \fIxhost\fP(1), \fIxfwp\fP(1),
 .I "Security Extension Specification"
 .PP
-Starting the server: \fIxdm\fP(1), \fIxinit\fP(1)
+Starting the server: \fIstartx\fP(1), \fIxdm\fP(1), \fIxinit\fP(1)
 .PP
 Controlling the server once started: \fIxset\fP(1), \fIxsetroot\fP(1),
 \fIxhost\fP(1)
 .PP
 Server-specific man pages:
-\fIXorg\fP(1), \fIXdmx\fP(1), \fIXnest\fP(1),
+\fIXorg\fP(1), \fIXdmx\fP(1), \fIXephyr\fP(1), \fIXnest\fP(1),
 \fIXvfb\fP(1), \fIXDarwin\fP(1), \fIXWin\fP(1).
 .PP
 Server internal documentation:
diff --git a/hw/kdrive/ephyr/Xephyr.man.pre b/hw/kdrive/ephyr/Xephyr.man.pre
index 7f0e811..f5e8b98 100644
--- a/hw/kdrive/ephyr/Xephyr.man.pre
+++ b/hw/kdrive/ephyr/Xephyr.man.pre
@@ -23,8 +23,8 @@ Xephyr - X server outputting to a window on a pre-existing X display
 .IR ... ]
 .SH DESCRIPTION
 .B Xephyr
-is a a kdrive server that outputs to a window on a pre-existing
-'host' X display.
+is a a kdrive server that outputs to a window on a pre-existing "host"
+X display.
 Think
 .I Xnest
 but with support for modern extensions like composite, damage and randr.
@@ -75,7 +75,7 @@ round it. It doesn't appear to break anything however.
 .IP \(bu 2
 Keyboard handling is basic but works.
 .TP \(bu 2
-Mouse button 5 probably wont work.
+Mouse button 5 probably won't work.
 .SH "SEE ALSO"
 X(__miscmansuffix__), Xserver(__appmansuffix__)
 .SH AUTHOR
-- 
1.5.6.5

